.github/workflows/plugin-check.yml

```
@@ -0,0 +1,36 @@
+    branches: [ main ]
+
+permissions:
+  contents: read
+
+jobs:
+  plugin-check:
+      runs-on: ubuntu-latest
+      timeout-minutes: 15
+      steps:
+      - uses: actions/checkout@v3
+            - name: Setup Composer Cache
```

actionlint (1.7.8)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)